python_docs_theme/layout.html

```diff
@@ -85,7 +85,10 @@ <h3>{{ _('Navigation') }}</h3>
         <span></span>
     </label>
     <nav class="nav-content" role="navigation">
-        <img src="{{ pathto('_static/' + 'py.svg', 1) }}" alt="Logo"/>
+         <a href="{{ pathto('index') }}">
```

Comment:
pathto(master_doc) would be more appropriate here?

Reply:
Or does CPython documentation use contents for the toctree?

Reply:
Thank you for such a quick review! :) I know what I added doesn't work for sure. Let me try pathto(master_doc).

Reply:
I need help! 
I don't know how to add the correct link to the `docs.python.org/3.9/`. 
In line 88 in layout.html I added `<a href="{{ pathto(theme_root_url) }}" `, and it creates `file:///.../doc-html(3)/3.9/https://www.python.org/.html`

Reply:
Don't wrap it in pathto, just `{{ theme_root_url }}` is enough.

That should fix it. :)

Reply:
Hmmm. It now links to python.org...

Reply:
That's what "root_url" in theme_config_options is set to in the conf.py file of the documentation being built.

Reply:
In other words, that is correct and I thought that's what you wanted?

Reply:
I guess I was confusing the two kind of links (to the docs home and the language/framework home). On the sites I used a lot lately (eg Vue.js) the style of the framework home page and the style of the documentation homepage are the same. And it feels like I just go to the documentation home, not the general home, when I click on the logo.

But you are right, now we have the correct behavior, linking to python.org.

We still don't have a way of going to the documentation homepage, `docs.python.org/3` from other pages on mobile. Is it necessary to add it, and add it in this PR specifically?

Reply:
If you want to have a link to the documentation homepage, that'd be `{{pathto(master_doc)}}` or `{{ pathto("index") }}`. No need for a |e on either of them.

Reply:
Honestly, this feels like something we can do in a follow up PR.

I'd really like to get this landed soon, so that we can start moving other things forward without needing to come back and update this.

Reply:
Please, let me know if I need to change anything? I would be happy to get this landed soon, too!